thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/ThreadView.kt

```diff
@@ -63,18 +65,81 @@ class ThreadView(
                 null,
             ),
         )
-        componentPopupMenu = FlatPopupMenu().apply {
-            add(
-                JMenu("Mark all with same...").apply {
-                    ThreadModel.ThreadColumns.forEach {col ->
-                        add(
-                            Action(col.header) {
-                                markAllWithSameValue(col)
-                            }
-                        )
+
+        fun markAllWithSameValue(property: Column<Thread, *>) {
+            val selectedPropertyValues = selectedRowIndices()
```

Comment:
This _could_ be simplified if single selection is guaranteed. But I don't think it matters that much